package.json

```diff
@@ -30,7 +30,7 @@
     "compile"
   ],
   "dependencies": {
-    "dustjs-linkedin": "^2.4.0",
+    "dustjs-linkedin": "^2.5.0",
```

Comment:
I am aware of how NPM handles versioning.

If a user has dustjs-linkedin@2.4 installed and then installs gulp-dust, if gulp-dust's version requirement is not changed, the existing peer dependency will be used. You get a tree like:

```
├── dustjs-linkedin@2.4.2
└─┬ gulp-dust@1.1.0
  ├── gulp-util@3.0.5
  └── through2@0.6.5
```

If the version requirement is changed, gulp-dust will get its own copy of dustjs-linkedin inside its own `node_modules` directory, allowing it to interoperate with an older version that is a peer dependency. You get a tree like:

```
├── dustjs-linkedin@2.3.6
└─┬ gulp-dust@1.1.0
  ├── dustjs-linkedin@2.7.1
  ├── gulp-util@3.0.5
  └── through2@0.6.5
```